database-commons/src/main/java/io/cdap/plugin/db/batch/sink/AbstractDBSink.java

```diff
@@ -279,44 +286,47 @@ private void setResultSetMetadata() throws Exception {
   }
 
   private void validateSchema(FailureCollector collector, Class<? extends Driver> jdbcDriverClass, String tableName,
-                              Schema inputSchema) {
+                              Schema inputSchema, String dbSchemaName) {
     String connectionString = dbSinkConfig.getConnectionString();
+    String fullyQualifiedTableName = dbSinkConfig.dbSchemaName == null ? dbSinkConfig.getEscapedTableName()
```

Comment:
similar comment here, already have these as params so don't need to again use config to access them.
```
String fullyQualifiedTableName = dbSchemaName == null ? dbSinkConfig.getEscapedTableName()
          : dbSchemaName + "." + dbSinkConfig.getEscapedTableName();
```